### PR TITLE
Add RTL layout support and Arabic localization

### PR DESCRIPTION
**Pull request by Rami-Sabbagh**
Hello 👋 

I've put some effort and added RTL support to the awesome theme you've have made.
And added Arabic localization.

I had to use a tailwind package to achieve that. And I did update the cached prebuilt css.

I've made that so I could use it for my own blog. And thought why not share it with everyone.

Please tell me what are your thoughts.

### CODE REVIEW

**Comment by Rami-Sabbagh** [comment]
A little note on defining RTL languages in `languages.yaml`: a `languageDirection: rtl` property should be added.

For example in Arabic:
```yaml
ar:
  languageCode: ar
  languageName: العربية
  languageDirection: rtl
  contentDir: content/ar
  title: مدوّنة
```

**Comment by wangchucheng** [comment]
Hi @Rami-Sabbagh,

Thank you so much for adding this awesome feature, it is quite necessary for Eureka to have this. Since I have no experiences in RTL languages, do you have a RTL example with Eureka and its source code? This may help me to review this PR. Thanks again for your contribution ;-)

**Comment by Rami-Sabbagh** [comment]
There's my personal blog, but the posts have not been translated to Arabic yet.
[AR](https://rami-sabbagh.github.io/ar/) • [EN](https://rami-sabbagh.github.io/) • [source-code](https://github.com/Rami-Sabbagh/rami-sabbagh.github.io/)

**Comment by Rami-Sabbagh** [comment]
I've done some modifications today to add singular & plural formatting support for the reading time metadata of each post.
By following the [hugo docs](https://gohugo.io/content-management/multilingual/#query-a-singularplural-translation).

Updated all the existing localizations to use the new property, but I'm afraid this might introduce backwards incompatible update. (Users will have to update their localization files).

File: i18n/en.yaml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 readMore:
   other: Read More
 readingTime:
-  other: '{{ .ReadingTime }} min read'
+  one: One min read
```
Comment (wangchucheng) [inline i18n/en.yaml:4]: I saw that `.Count` is a number instead of a work, would it be better if we use 1 instead of one?

**Review by wangchucheng — COMMENTED** [review]
Hi @Rami-Sabbagh,

Thanks for your contribution! I just have one comment as above, feel free to talk to me if you have any thoughts.

File: i18n/en.yaml (hunk 1 above)
Comment (Rami-Sabbagh) [reply i18n/en.yaml:4]: Surely we can

File: i18n/en.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 readMore:
   other: Read More
 readingTime:
-  other: '{{ .ReadingTime }} min read'
+  other: '{{ .Count }} mins read'
```
Comment (wangchucheng) [inline i18n/en.yaml:4]: Maybe we can keep the `one` part and have something like `one: '1 min read'`?

**Review by wangchucheng — COMMENTED** [review]
Hi @Rami-Sabbagh,

Just one little comment about the `one` in i18n. And if you are cool with the whole pr, I can merge it later

File: i18n/en.yaml (hunk 2 above)
Comment (Rami-Sabbagh) [reply i18n/en.yaml:4]: There you go

**Comment by Rami-Sabbagh** [comment]
I think the RTL support is good to go now.
For any other tweaks I might be doing in my fork, will wait few weeks before opening a new PR so you can review them all at once.

Although I have one question: Can we have the footer content localized? (the one which includes the hugo-eureka link)

**Comment by wangchucheng** [comment]
> Although I have one question: Can we have the footer content localized? (the one which includes the hugo-eureka link)

That's a good idea and I think it is possible. I will look into that later ;-)

**Comment by wangchucheng** [comment]
Just one quick remainder for your further development, you need to set hugo to production in order to purge the css if you want to update the cached resources. I found that the cached assets was like 4.4mb, but after purging, it would be around 20-25kb. I have fixed this issue so don't worry about it.

**Pull request merged**